spec/models/aliquot_spec.rb

```diff
@@ -194,4 +194,73 @@
       expect(described_class.publishable.count).to eq(10)
     end
   end
+
+  context 'sample sheet behaviour' do
```

Comment:
Should `tagged?` tests and `collection?` tests be moved into this context?

Missing tests for tag_set.